### DISCUSSION TITLE
Avoid click for M2M in tables by simplifying

### DISCUSSION THREAD

**Discussion by pierre-H** [directus/directus#5957] [Feature Requests]
Sometimes in M2M, you need to click to see the value.
In this example, I have to click 2 times :

![image](https://user-images.githubusercontent.com/6079305/120164030-6b3bd480-c1fa-11eb-93b4-257d5318490c.png)

I believe that when there is only 1 item, the value should be automatically displayed.

**Comment by philleepflorence** [comment]
It just so happens I went through this some hours ago.

It looks like the display  for the **M2M** and **('o2m', 'm2m', 'm2a', 'translations', 'files')** Interfaces, also serves as navigation to the related item(s).

I can see the value of displaying a single item where applicable.
I guess how to update the display and accommodate the interfaces and data-types might be a starting question?

[Related Values Display: Directus Github](https://github.com/directus/directus/blob/main/app/src/displays/related-values/related-values.vue)
[Displays: Directus Docs](https://docs.directus.io/concepts/displays/#displays)

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
